contrib/externs/showdown-1.3.js

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Missing the Apache License header. You can copy it from the other externs in the directory.